monocle/static/js/main.js

```diff
@@ -357,8 +357,23 @@ function getWorkers() {
     });
 }
 
-var map = L.map('main-map', {preferCanvas: true}).setView(_MapCoords, 13);
+var params = {};
+window.location.href.replace(/[?&]+([^=&]+)=([^&]*)/gi, function(m, key, value) {
+  params[key] = parseFloat(value);
+});
+
+var parsezoom = 13;
+if(parseFloat(params.zoom)) parsezoom = params.zoom;
 
+if(parseFloat(params.lat) && parseFloat(params.lon)){
+    var map = new L.Map('main-map', {
+        center: [params.lat, params.lon], 
```

Comment:
ok, I see now, I still think it would be better to use `parseFloat` where it is needed and not above,
because if now someone desides that he wants to use params the `parseFloat` may be removed again as the required param shouldn't be a float